angular-app/src/app/create-project-dialog/create-project-dialog.component.ts

```diff
@@ -40,6 +41,9 @@ export class CreateProjectDialogComponent implements OnInit {
       title: this.title,
       roles: roles,
       tags: this.tags,
+      files: [],
+      textDoc: this.lyricDoc,
+      docInfo: ""
```

Comment:
I think it might be slightly clearer if these were renamed to indicate what is the file name vs the contents of the file. E.g. `lyricsFileName`, `lyricsContent`